src/elements/common/annotator-context/withAnnotations.tsx

```diff
@@ -20,6 +21,7 @@ export default function withAnnotations<P extends object>(
         static displayName: string;
 
         state: AnnotatorState = {
+            activeAnnotationId: null,
             annotation: undefined,
```

Comment:
You should be able to omit these entirely if you add `?` to these properties in the AnnotatorState type.